### ISSUE TITLE
LYWSD03MMC unvailable

### ISSUE THREAD

**Issue by patoche83** [custom-components/ble_monitor#207]
Hi, firstly thanks for this custom component and the latest changes/improvement. 
I was using mitemp_bt since until few days with 4x LYWSD03MMC sensors. Then I have switched to the passive ble monitor.
I took the occasion to also update to the latest HA and supervisor version. After instal and setting up the passive BLE monitor and the sensors, I had sensor state = unavailable. 
configuration yaml does not contains anything now about mitemp_ble

I have the following setup:
## System Health

version | 2020.12.1
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.6
os_name | Linux
os_version | 5.4.79-v7
arch | armv7l
timezone | Europe/Paris

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4546
Installed Version | 1.9.0
Stage | running
Available Repositories | 753
Installed Repositories | 23

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 13 janvier 2021 à 1:00
relayer_connected | true
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | false
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Hass.io</summary>

host_os | Home Assistant OS 5.8
-- | --
update_channel | stable
supervisor_version | 2020.12.7
docker_version | 19.03.13
disk_total | 28.6 GB
disk_used | 12.4 GB
healthy | true
supported | true
board | rpi3
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.2.0), Samba share (9.3.0), Mosquitto broker (5.1), SSH & Web Terminal (7.8.0), AppDaemon 3 (5.0.1), ESPHome (1.15.3), Grafana (5.2.3), Check Home Assistant configuration (3.6.0), AdGuard Home (2.6.1), Node-RED (7.2.10)

</details>

<details><summary>Lovelace</summary>

dashboards | 3
-- | --
mode | storage
views | 8
resources | 20

</details>

logs from HA core:

ERROR (Thread-110) [custom_components.ble_monitor] HCIdump thread: Runtime error while stop scan request on hci0: Event loop stopped before Future completed.

and

ERROR (Thread-110) [custom_components.ble_monitor] HCIdump thread: Runtime error while sending scan request on hci0: Event loop stopped before Future completed.

and from host logs a lot of the following one:
[169683.011172] Bluetooth: hci0: Frame reassembly failed (-84).

I have connected a keyboard and a screen to mi rpi3b, and played a bit with BTmon and hcitools. Honnestly I did not recall how I have been able to get to this command and also where to enter the famous command "sudo setcap... to give python access to the BT stack. I am not an expert on this. I did not noticed something strange. BT is on hci0.
After restarting HA, I got all sensors available and working. :) I have then added ZHA integration using a deconz stick. So I have restarted HA and after this restart, BLE sensors are unavailable again :( 

could it be that python does not have access to the BT stack ? cf.  install instructions of the passive BLE monitor regarding rootless access.
BTW, how to launch BTmon and hcitools from the ssh terminal within HA  ?

Thanks for any help you could provide.

**Comment by Ernst79** [comment]
I assume the Runtime Error is gone now, is that right? Can you double check the hci number again with

```
hcitool dev
```

**Comment by Magalex2x14** [comment]
In HassOS, to work with system utilities, you need to access the host system. This is described [here](https://developers.home-assistant.io/docs/operating-system/debugging/#ssh-access-to-the-host).
In general, no additional actions are required in the official HassOS build - everything should work out of the box. The errors you mentioned in the system log are alarming, but do not necessarily indicate problems in the operation of the bluetooth subsystem - I sometimes see such errors in myself, but I do not notice any effect on the operation of integration and the continuity of data reception. In addition, I still haven't heard of any compatibility issues associated with using ZigBee sticks. But perhaps this is something... I don’t know how to help you yet...

**Comment by patoche83** [comment]
Hi Guys, thank you for taking time to look into this.
Ernst, the runtime error, you mean HCIDump I guess was appearing repeatedly not only once after startup.
Using the SSH & Web Terminal Add on, hcitool is not available, I have used bluetoothctl and got this error 
"Failed to start discovery: org.bluez.Error.NotReady". playing a bit with the different commands, like power off/on, scan on etc.. did not help. But then I reminded that when passive BLE monitor and sensor where working, I had my Zigbee usb stick (an old dresden elektronik one ;) ) not connected. So this morning I have unplugged it and restarted HA and tada !! BLE was working again. So I have moved the usb stick to another usb port and now all seems ok after several restart.
By looking into this, looks like it is a conflict between the UART used to communicate with the BLE chipset and the serial over usb used by the stick or something close to this.
Alex, thanks for the link. 
Anyway thanks for stopping by my issue and trying to help :)
